docs-source/content/userguide/managing-domains/model-in-image/usage.md

```diff
@@ -48,11 +48,11 @@ There are two methods for layering Model in Image artifacts on top of a base ima
 
 For more information about model file syntax, see [Model files]({{< relref "/userguide/managing-domains/model-in-image/model-files.md" >}}).
 
-#### 3. Optional WDT model config map
+#### 3. Optional WDT model configmap
```

Comment:
i found line 40 confusing - it implied to me that i do not have to put the archive in the image, that i can put it somewhere else - maybe in the configmap, like this section goes on to describe for the model and properties.  but i think it actually means to say that is is optional because i might not have one at all? is that right?

Reply:
good catch - i've updated this and the related mention in the sample doc